tests/test_modeling_roberta.py

```diff
@@ -527,3 +529,20 @@ def test_inference_classification_head(self):
         # expected_tensor = roberta.predict("mnli", input_ids, return_logits=True).detach()
 
         self.assertTrue(torch.allclose(output, expected_tensor, atol=1e-4))
+
+    # XXX: this might be a candidate for common tests if we have many of those
+    def test_lm_head_ignore_keys(self):
+        keys_to_ignore_on_save_tied = [r"lm_head.decoder.weight", r"lm_head.decoder.bias"]
+        keys_to_ignore_on_save_untied = [r"lm_head.decoder.bias"]
+        config = RobertaConfig.from_pretrained(ROBERTA_TINY)
+        for cls in [RobertaForMaskedLM, RobertaForCausalLM]:
+            model = cls.from_pretrained(ROBERTA_TINY)
```

Comment:
This could also be `cls(config)` which is already instantiated with a tiny configuration - or do you need to explicitly use the weights in `ROBERTA_TINY`?

Reply:
It was because it was a different config. I recoded it to make it more readable. Thank you for pointing this out, @LysandreJik